design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,197 @@
+    ): Returns @doc(description: "Information about the order returns.")
+    items_eligible_for_return: [OrderItemInterface] @doc(description: "A list of order items eligible for return.")
+}
+
+type Returns {
+    items: [Return] @doc(description: "List of returns")
+    page_info: SearchResultPageInfo @doc(description: "Pagination metadata")
+    total_count: Int @doc(description: "Total count of customer returns")
+}
+
+type Return @doc(description: "Customer return") {
+    id: ID! @doc(description: "Return number")
+    order_id: String! @doc(description: "ID of the order based on which the return was created.")
+    creation_date: String! @doc(description: "The date when the return was requested.")
+    customer_email: String! @doc(description: "Email of the customer who created the return.")
+    customer_name: String @doc(description: "The name of the person returning the item")
+    status: ReturnStatus @doc(description: "Return status.")
+    shipping: ReturnShipping @doc(description: "Shipping information for the return.")
+    comments: [ReturnComment] @doc(description: "A list of comments posted for the return.")
+    items: [ReturnItem] @doc(description: "A list of items being returned.")
+    available_shipping_carriers: [ReturnShippingCarrier] @doc(description: "A list of shipping carriers available for returns.")
+}
+
+type ReturnItem {
+    id: ID!
+    product: ProductInterface!
+    custom_attributes: [CustomAttribute]
+    request_quantity: Float!
+    quantity: Float!
+    status: ReturnItemStatus!
+}
+
+# See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/custom-attributes-container.md
+type CustomAttribute {
+    id: ID!
+    label: String!
+    value: String! @doc(description: "JSON encoded value of the attribute.")
+}
+
+type ReturnComment {
+    id: ID! @doc(description: "Comment ID.")
+    created_by: String! @doc(description: "The name of the user who created posted the comment.")
+    created_at: String! @doc(description: "The date and time when the comment was posted.")
+    text: String! @doc(description: "The comment text.")
+}
+
+type ReturnShipping {
+    address: ReturnShippingAddress @doc(description: "Return shipping address, which is specified by the admin.")
+    tracking(id: ID): [ReturnShippingTracking] @doc(description: "Tracking information for all or a single tracking record when ID is provided.")
+}
+
+type ReturnShippingCarrier {
+    id: ID!
+    label: String!
+}
+
+type ReturnShippingTracking {
+    id: ID!
+    carier: ReturnShippingCarrier!
+    tracking_number: String!
+    status_type: ReturnShippingTrackingStatusType!
+    status: String!
+}
+
+enum ReturnShippingTrackingStatusType {
+    INFORMATION
+    ERROR
+}
+
+type ReturnShippingAddress {
+    contact_name: String
+    street: [String]!
+    city: String!
+    region: Region!
+    postcode: String!
+    country: Country!
+    telephone: String
+}
+
+enum ReturnStatus {
```

Comment:
There's no `PARTIALLY_REJECTED` or `PARTIALLY_APPROVED` in the devdocs - is this a case of the docs being wrong or the schema being wrong (legit question, I have no clue)?

https://docs.magento.com/user-guide/sales/rma-lifecycle.html#rma-status

Reply:
Yeah, good observation. I found these in the codebase and have already reached out to product and docs team to verify and correct the docs. If the docs are right - they should reach back to us to fix the schema

Reply:
Adding link for reference: https://github.com/magento/magento2ee/blob/2f1f767fbaeb4b9abc0cbad7fecbc1f070b2ce7f/app/code/Magento/Rma/Model/Rma/Source/Status.php#L68-L98

I think the docs are wrong